build/linux_dist.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
 mkdir dist
-for arch in "amd64"
+for arch in "amd64" "arm64"
 do
+  unset -v CC
```

Comment:
So in case of amd64, will the `CC` be unset and not assigned with value again, so we cannot find `CC`?

Reply:
Compiler will use a default value for CC if not found. I have tested in both CI and local environments.

Reply:
Cool LGTM then. Although I would prefer moving this line into the `if then` to make it really clear.